src/languages/en.js

```diff
@@ -846,4 +846,13 @@ export default {
         screenShare: 'Screen share',
         screenShareRequest: 'Expensify is inviting you to a screen share',
     },
+    genericErrorPage: {
+        title: 'Uh-oh, something went wrong!',
+        body: {
+            helpTextMobile: 'Please try closing and reopening the app or switching to ',
```

Comment:
Don't include leading or trailing spaces in translations:

```suggestion
            helpTextMobile: 'Please try closing and reopening the app or switching to',
```